data/roles/win2022642009azurevs.yaml

```diff
@@ -0,0 +1,55 @@
+---
+
+win-worker:
+  # generic-worker
+  generic_worker:
+    # File versions
+    name: "generic-worker-multiuser-windows-amd64"
+    version: '53.2.1'
```

Comment:
Did you want to bump this to `54.4.1`? That's what I'm rolling out with windows cloud workers (and hardware).

Reply:
good call.